modelscope_agent/storage/base.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class BaseStorage(ABC):
+
+    @abstractmethod
```

Comment:
where is the init interface? some storage need to be initialized with some resources

Reply:
init method might be varied from one db to other dbs. 
no common attribute at this moment.